l2-contracts/contracts/ConsensusRegistry.sol

```diff
@@ -111,19 +115,21 @@ contract ConsensusRegistry is IConsensusRegistry, Initializable, Ownable2StepUpg
 
         emit NodeAdded({
             nodeOwner: _nodeOwner,
+            isValidatorActive: _isValidatorActive,
             validatorWeight: _validatorWeight,
             validatorPubKey: _validatorPubKey,
             validatorPoP: _validatorPoP,
+            isAttesterActive: _isAttesterActive,
             attesterWeight: _attesterWeight,
             attesterPubKey: _attesterPubKey
         });
     }
 
-    /// @notice Deactivates a node, preventing it from participating in committees.
+    /// @notice Deactivates an attester, preventing it from participating in attester committees.
     /// @dev Only callable by the contract owner or the node owner.
     /// @dev Verifies that the node owner exists in the registry.
     /// @param _nodeOwner The address of the node's owner to be inactivated.
-    function deactivate(address _nodeOwner) external onlyOwnerOrNodeOwner(_nodeOwner) {
+    function deactivate_attester(address _nodeOwner) external onlyOwnerOrNodeOwner(_nodeOwner) {
```

Comment:
We use camelCase in Solidity contracts. Please fix here and in other places
```suggestion
    function deactivateAttester(address _nodeOwner) external onlyOwnerOrNodeOwner(_nodeOwner) {
```

Reply:
True. Muscle memory from Rust. xD